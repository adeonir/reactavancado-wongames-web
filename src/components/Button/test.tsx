import { screen } from '@testing-library/react'
import { renderWithTheme } from 'utils/tests/helpers'

import { Button } from '.'

describe('<Button />', () => {
  it('should render the button', () => {
    const {
      container: { firstChild },
    } = renderWithTheme(<Button>Buy now</Button>)
    expect(screen.getByRole('button', { name: /buy now/i })).toBeInTheDocument()
    expect(firstChild).toMatchSnapshot()
  })
})
