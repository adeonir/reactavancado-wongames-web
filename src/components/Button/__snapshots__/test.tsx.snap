// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render the button 1`] = `
.c0 {
  background: linear-gradient(180deg,#ff5f5f 0%,#f062c0 50%);
  color: #fafafa;
  border-radius: 0.4rem;
  border: 0;
  font-size: 1.4rem;
  padding: 0.8rem 2.4rem;
  height: 4rem;
}

<button
  class="c0"
>
  <span>
    Buy now
  </span>
</button>
`
